source/cluster-man/doveadm-cluster.1.rst.in

```diff
@@ -0,0 +1,58 @@
+.. _man-1_doveadm-cluster:
+
+===============
+doveadm-cluster
+===============
+
+NAME
+====
+
+doveadm-cluster - Utility actions for Dovecot cluster
+
+SYNOPSIS
+========
+
+**doveadm** [**-Dv**] [**-f** *formatter*] cluster *command* [*arguments*]
+
+DESCRIPTION
+===========
+
+**doveadm cluster** can be used to run actions that interact with and
+modify Dovecot cluster.
+
+.. include:: global-options-formatter.inc
+
+COMMANDS
+========
+
+**doveadm cluster site**
+  :man:`doveadm-cluster-site(1)`, Site management commands
+
+**doveadm cluster backend**
+  :man:`doveadm-cluster-backend(1)`, Backend management commands
+
+**doveadm cluster group**
+  :man:`doveadm-cluster-group(1)`, Group management commands
+
+**doveadm cluster user**
+  :man:`doveadm-cluster-user(1)`, User management commands
+
+**doveadm cluster geodb**
+  :man:`doveadm-cluster-geodb(1)`, GeoDB management commands
+
+**doveadm cluster localdb**
+  :man:`doveadm-cluster-localdb(1)`, LocalDB management commands
+
+**doveadm cluster notify**
+  :man:"doveadm-cluster-notify(1)", 
```

Comment:
localdb & notify are hidden, remove?

Reply:
there is comment in the jira ticket about moving notifys into user and group categories.

Reply:
i removed notify at least.